.github/workflows/beam_LoadTests_Python_CoGBK_Dataflow_Batch.yml

```diff
@@ -79,10 +79,10 @@ jobs:
           test-type: load
           test-language: python
           argument-file-paths: |
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_100b_Single_Key.txt
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_100b_Multiple_Keys.txt
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_10kB.txt
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_2MB.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_100b_Single_Key.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_100b_Multiple_Keys.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_10kB.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_2MB.txt
```

Comment:
What was the issue we're fixing here? Are paths too long?

I like the per directory organization, if file length is the issue maybe we could do something like `${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/dataflow_Batch_2MB.txt` which truncates to basically the same length

Reply:
Yes, I corrected the file path to the correct one, following the same approach we use for all other jobs.

Reply:
Ok, I think it makes sense to attach to the consistent pattern, so I'll merge. In the future we may consider nesting these hierarchically